packages/ethereum-contracts/contracts/mocks/SuperTokenMock.sol

```diff
@@ -59,8 +59,8 @@ contract SuperTokenStorageLayoutTester is SuperToken {
         require (slot == 18 && offset == 0, "_operators changed location");
         // uses 4 slots
 
-        assembly { slot:= _reserve22.slot offset := _reserve22.offset }
-        require (slot == 22 && offset == 0, "_reserve22 changed location");
+        assembly { slot:= _reserve23.slot offset := _reserve23.offset }
```

Comment:
`_adminOverride` field should be tested.

Reply:
good catch 🔍